src/classes/CRLP_ApiService_TEST.cls

```diff
@@ -35,6 +35,10 @@
 */
 @IsTest(IsParallel=true)
 private class CRLP_ApiService_TEST {
+    private static UTIL_IntegrationConfig_TEST.MockPackage mockPackage;
+    private static MockCallable callableMock;
+
+    private static final String ROLLUP_DEFINITIONS = '[{"useFiscalYear":false,"timeBoundOperation":"All_Time","theSum":0,"theCount":0,"textValue":[null,null,null,null],"summaryObject":"General_Accounting_Unit__c","sumByYear":{}}]';
```

Comment:
This is ok as long as this is not going to be re-cast as `List<CRLP_Rollup>`.

Reply:
I'll double check - if it is, I'll make a similar fix to as what I did in the child branch